### ISSUE TITLE
Folder path length exceeded on Windows in Examples subdirectory

### ISSUE THREAD

**Issue by timkrins** [js-next/react-style#130]
Installing via npm, the path lengths in the examples directory are very close to exceeding or exceed the Windows limits. For example:

`C:\a\node_modules\react-style\examples\0.13\server\node_modules\webpack\node_modules\watchpack\node_modules\chokidar\node_modules\anymatch\node_modules\micromatch\node_modules\regex-cache\node_modules\benchmarked\node_modules\file-reader\node_modules\map-files\node_modules\globby\node_modules\array-union\node_modules\array-uniq\package.json`

is 340 characters when "un-windows-truncated" and 271 characters when truncated:

`C:\a\NODE_M~1\REACT-~4\examples\0.13\server\NODE_M~1\webpack\NODE_M~1\WATCHP~1\NODE_M~1\chokidar\NODE_M~1\anymatch\NODE_M~1\MICROM~1\NODE_M~1\REGEX-~1\NODE_M~1\BENCHM~1\NODE_M~1\FILE-R~1\NODE_M~1\MAP-FI~1\NODE_M~1\globby\NODE_M~1\ARRAY-~2\NODE_M~1\array-uniq\package.json`

I know its a npm/node problem with this particular directory structure, leaving here in case people are having issues.

Linked to npm/npm#5458 and nodejs/node-v0.x-archive#6960

**Comment by andreypopp** [comment]
Should be solved with npm v3, no?

**Comment by timkrins** [comment]
No, unfortunately not - tested with npm@3.3.4 and node@4.1.1 locally.

It looks like this folder structure is not created by your locally running NPM - they have been hard-baked into the react-style release.

Maybe react-style's examples directory structure just needs to be updated with an NPM structure created from npm >= v3 ?
